### PR TITLE
Display Resource metadata in Editor

### PR DESCRIPTION
**Pull request by dhaneshnm**
- Add a toggle to show/hide meta data in editor.
- Fetch and display meta-data, the content is non editable when meta
data is displayed.

Implements, BlueBrain/nexus/issues/779

### CODE REVIEW

**Comment by dhaneshnm** [comment]
Screenshot_2019-11-04 A knowledge graph for data-driven science(2)
Screenshot_2019-11-04 A knowledge graph for data-driven science(1)
Screenshot_2019-11-04 A knowledge graph for data-driven science

File: src/shared/components/ResourceEditor/index.tsx [hunk 1]
```diff
@@ -90,6 +94,14 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
               onChange={onFormatChange}
             />
           )}
+          {!expanded && !isEditing && valid && (
+            <Switch
```
Comment (kenjinp) [inline src/shared/components/ResourceEditor/index.tsx]: I would put this switch before the expanded toggle, so the expanded toggle doesn't get pushed around if this one isn't moved. 

Also metadata is used as one word in or documentation, so we should do the same here. We can also be more consistent with our copy, using lowercase on all the button toggles here.

Comment (kenjinp) [reply src/shared/components/ResourceEditor/index.tsx]: Also, there should be a space between the toggles, like so `{" " }`

File: src/shared/containers/ResourceEditor.tsx [hunk 2]
```diff
@@ -23,14 +24,16 @@ const ResourceEditorContainer: React.FunctionComponent<{
...
+  
   const {
     orgLabel,
     projectLabel,
     resourceId,
   } = getResourceLabelsAndIdsFromSelf(self);
   const [{ busy, resource, error }, setResource] = React.useState<{
     busy: boolean;
-    resource: ResourceSource | ExpandedResource | null;
+    resource: ResourceSource | ExpandedResource | Resource |null;
```
Comment (Stafeeva) [inline src/shared/containers/ResourceEditor.tsx]: here should be a space after the pipe: `Resource | null`;

File: src/shared/containers/ResourceEditor.tsx [hunk 3]
```diff
@@ -43,6 +46,23 @@ const ResourceEditorContainer: React.FunctionComponent<{
     setExpanded(!expanded);
   };
 
+  const handleMetaDataChange = () => {
+    setShowMetaData(!showMetaData);
+  };
+
+  const getNewResource = async () => {
+    if(expanded) {
```
Comment (Stafeeva) [inline src/shared/containers/ResourceEditor.tsx]: please add a space after `if` so it is consistent

File: src/shared/containers/ResourceEditor.tsx [hunk 4]
```diff
@@ -43,6 +46,23 @@ const ResourceEditorContainer: React.FunctionComponent<{
...
+      return  await nexus.Resource.get(orgLabel, projectLabel, resourceId, {
+        rev,
+        format: 'expanded',
+      })
+    } 
+    if(showMetaData) {
```
Comment (Stafeeva) [inline src/shared/containers/ResourceEditor.tsx]: same as above

File: src/shared/containers/ResourceEditor.tsx [hunk 5]
```diff
@@ -54,12 +74,8 @@ const ResourceEditorContainer: React.FunctionComponent<{
...
           busy: true,
         });
-        const newResource = expanded
-          ? await nexus.Resource.get(orgLabel, projectLabel, resourceId, {
-              rev,
-              format: 'expanded',
-            })
-          : await nexus.Resource.getSource(orgLabel, projectLabel, resourceId);
+
+        const  newResource =  await getNewResource();
```
Comment (Stafeeva) [inline src/shared/containers/ResourceEditor.tsx]: I think there are extra spaces here after `const` and `await` :)

File: src/shared/components/ResourceEditor/index.tsx (hunk 1 above)
Comment (dhaneshnm) [reply src/shared/components/ResourceEditor/index.tsx]: Updated the code.

File: src/shared/containers/ResourceEditor.tsx (hunk 5 above)
Comment (dhaneshnm) [reply src/shared/containers/ResourceEditor.tsx]: Ran prettier. They should be good now.

**Pull request merged**